packages/shared/lib/core/utils/format.utils.ts

```diff
@@ -0,0 +1,22 @@
+import { HEXADECIMAL_PREFIX, HEXADECIMAL_REGEXP } from './constants'
+
+export function formatHexString(
+    hex: string,
+    shouldBeUppercase: boolean = true,
+    shouldIncludePrefix: boolean = true
+): string {
+    if (!hex || typeof hex !== 'string' || !HEXADECIMAL_REGEXP.test(hex)) {
```

Comment:
No need for `typeof hex !== 'string'`. You can expect a string here, since you type it as an argument here. If you supply something else, TSC will scream at you!